File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/WaitStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;


import com.mokatest.platform.demos.domain.ui.uiEnum.WaitType;
import lombok.Data;

// 等待操作
@Data
public class WaitStepDTO extends StepBaseDTO {

    private String stepType;
    // todo 考虑做等待类型
    private WaitType waitType;
    private Integer waitTime; // 等待时间
}

File: frontEnd/src/types/dto/stepDTO/WaitStepDTO.ts
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {WaitType} from "@/types/enum/WaitType";

// 等待操作
export class WaitStepDTO extends StepBaseDTO {

    stepType: string = 'WAIT';

    // 等待类型
    waitType: WaitType | string = 'TIME';
    waitTime: number = 0 // 等待时间
}